src/app/app/form/layout.tsx

```diff
@@ -0,0 +1,20 @@
+import { Metadata } from "next";
+import { PropsWithChildren } from "react";
+import { getServerSession } from "next-auth";
+import { redirect } from "next/navigation";
+import { authOptions, makeRedirectURL, getOriginPath } from "@/shared/auth";
+
+export const metadata: Metadata = {
+  title: "PodCodar",
+};
+
+export default async function RootLayout({ children }: PropsWithChildren) {
```

Comment:
We can remove this file, as we already have this logic on `/src/app/app/layout.tsx`